### ISSUE TITLE
Crash when long press on erase button with SwiftKey

### ISSUE THREAD

**Issue by SithEngineer** [splitwise/TokenAutoComplete#160]
same as [#73](https://github.com/splitwise/TokenAutoComplete/issues/73) but with a new stacktrace in version 2.0.1:
```
java.lang.IndexOutOfBoundsException: charAt: 0 >= length 0
            at android.text.SpannableStringBuilder.charAt(SpannableStringBuilder.java:116)
            at com.tokenautocomplete.TokenCompleteTextView$TokenTextWatcher.afterTextChanged(TokenCompleteTextView.java:1184)
            at android.widget.TextView.sendAfterTextChanged(TextView.java:7679)
            at android.widget.TextView$ChangeWatcher.afterTextChanged(TextView.java:9451)
            at android.text.SpannableStringBuilder.sendAfterTextChanged(SpannableStringBuilder.java:972)
            at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:516)
            at android.text.SpannableStringBuilder.delete(SpannableStringBuilder.java:216)
            at android.text.SpannableStringBuilder.delete(SpannableStringBuilder.java:33)
            at android.view.inputmethod.BaseInputConnection.deleteSurroundingText(BaseInputConnection.java:244)
            at android.view.inputmethod.InputConnectionWrapper.deleteSurroundingText(InputConnectionWrapper.java:66)
            at com.tokenautocomplete.TokenCompleteTextView$TokenInputConnection.deleteSurroundingText(TokenCompleteTextView.java:1360)
            at com.android.internal.view.IInputConnectionWrapper.executeMessage(IInputConnectionWrapper.java:389)
            at com.android.internal.view.IInputConnectionWrapper$MyHandler.handleMessage(IInputConnectionWrapper.java:78)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5254)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:898)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:693)
```

**Comment by SithEngineer** [comment]
The previous stacktrace is shown in version 2.0.1

**Comment by NileshJarad** [comment]
I am also facing same issue

**Comment by mgod** [comment]
I believe I've fixed this in c60d372a45f9197e98a83c7fc5aa7ad1e8d949ce. I expect to have a release with this fix out in the next few days.
